src/molecules/end/Comparator.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { Icon } from 'src/utils/const';
+import { Chip, ChipProps } from 'src/molecules/Chip';
+import { layoutStyles } from 'src/utils/styles';
+import { registeredTextStyle } from 'src/utils/textStyles';
+
+const styles = StyleSheet.create({
+  Wrapper: {
+    height: 72,
+    ...layoutStyles.ColumnJustifyStart,
+    alignItems: 'flex-end',
+    width: '40%',
+  },
+  Title: registeredTextStyle('titleBold'),
+  Content: { marginTop: 8 },
+  BottomPanel: {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'nowrap',
+    marginTop: 8,
+  },
+  ComparatorBodyMargin: {
+    flexShrink: 1,
+    marginRight: 2,
+  },
+});
+
+export type EndComparatorProps = {
+  title: string;
+  content?: ChipProps;
+  bottomLabel?: EndComparatorLabelType;
+};
+
+type EndComparatorLabelType = {
+  icons?: IconType[];
+  text?: string;
+};
+
+type IconType = {
+  name: string;
+  color: string;
+};
+
+const renderIcon = (icon: IconType, index: number) => {
+  return (
+    <View key={index} style={styles.ComparatorBodyMargin}>
+      <Icon name={`${icon.name}-small`} color={icon.color} size={20} />
+    </View>
+  );
+};
+
+const renderText = (textBody: string) => {
+  return (
+    <Text
+      numberOfLines={1}
+      style={{
+        ...registeredTextStyle('body'),
+        ...styles.ComparatorBodyMargin,
```

Comment:
this, on the other hand, is called with every render and should better be moved to the `StyleSheet.create()` call

Reply:
Thanks and Sorry. I fixed this in several files with commit https://github.com/urbi-mobility/react-native-urbi-ui/pull/17/commits/b36a6b1050dc80c338a0fed331c9bf30a86823f8